rke/k8s_rke_system_images.go

```diff
@@ -6086,6 +6086,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.18.0",
 		},
+		// Enabled in v2.6.3
+		"v1.22.3-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.0",
+			Kubernetes:                "rancher/hyperkube:v1.22.3-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.78",
+			NginxProxy:                "rancher/rke-tools:v0.1.78",
+			CertDownloader:            "rancher/rke-tools:v0.1.78",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.78",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.17.4",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.17.4",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.17.4",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.14.0",
```

Comment:
We should bump the flannel version here since we're planning to merge the flannel bumps to prior k8s releases as part of a direct OOB release to release-v2.6 / dev-v2.6 in https://github.com/rancher/kontainer-driver-metadata/pull/736, which means that there won't be another PR (to my knowledge) where we plan to bump Flannel in `dev-v2.6`.

```suggestion
			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
```

Reply:
This is required for supporting Flannel in CentOS 8 nodes that may encounter an iptables segfault.

Reply:
This will be addressed in https://github.com/rancher/kontainer-driver-metadata/pull/739